styles.css

```diff
@@ -0,0 +1,183 @@
+.wrapper{
+  margin: auto;
+  width: 80%;
+}
+.header{
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 100px;
+}
+.logo{
+  position: relative;
+  padding: 40px 0;
+  padding-left: 44px;
+  font: bold 24px/31px 'PT Serif',serif;
+}
+.logo__circle{
+  position:absolute;
+  left: 0;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 28px;
+  height: 28px;
+  border-radius: 50%;
+  opacity: .7;
+}
+.logo__circle_color_blue{
+  left: 8px;
+  top: 56%;
+  background: #C8DBFB;
+}
+.logo__cirle_color_dark-blue{
+  background: #0056EC;
```

Comment:
Codacy found an issue: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/site99/pullRequest?prid=6205931)